Kudu.Contracts/Jobs/JobSettings.cs

```diff
@@ -31,6 +31,13 @@ public bool IsSingleton
                 return GetSetting(JobSettingsKeys.IsSingleton, false);
             }
         }
+        public bool IsWebHook
```

Comment:
Agree that the naming of this should be generalized. Who sets the setting, and how is it used?